_data/developer.yml

```diff
@@ -348,6 +348,16 @@ websites:
       - hardware
     doc: https://help.ubuntu.com/community/SSO/FAQs/2FA
 
+  - name: LogicMonitor
+    url: https://logicmonitor.com
+    img: logicmonitor.svg
+    tfa:
+      - totp
```

Comment:
```suggestion
```
I don't think regular `totp` is available here - you _have_ to use Authy, it seems.